Lsof.8

```diff
@@ -4457,6 +4453,7 @@ is the first component of the host's name returned by
 .SH AUTHORS
 .I Lsof
 was written by Victor A.\&Abell <abe@purdue.edu> of Purdue University.
+Since version 4.93.0, lsof-org at GitHUB team maintains.
```

Comment:
Slightly better English would be:

Since version 4.93.0, the lsof-org team at GitHub is maintaining 
.I Lsof

Reply:
Thank you, I will update the change. BTW, cirrus-ci for freebsd13 is failed for this PR.

Reply:
Looks like a flake on cirrus.  I requested a re-run.  Will check back in a bit.

Reply:
The re-run was successful (look further down in the list).

Reply:
Oh, I see.